components/elm/src/biogeophys/BandDiagonalMod.F90

```diff
@@ -218,4 +218,1890 @@ subroutine BandDiagonal(bounds, lbj, ubj, jtop, jbot, numf, filter, nband, b, r,
 
   end subroutine BandDiagonal
 
+
+
+
+
+      SUBROUTINE DCOPY(N,DX,INCX,DY,INCY)
```

Comment:
Preface this code with a comment about its source (the URL I gave you) and a link to this license: https://netlib.org/lapack/LICENSE.txt